packages/gafl-webapp-service/src/pages/multibuy/change-licence-options/__tests__/route.spec.js

```diff
@@ -0,0 +1,391 @@
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.isContinuing).toBeTruthy()
+    })
+
+    it('should return isContinuing as false if both renewedEndDate is not equal to licenceStartDate', async () => {
+      const request = {
+        cache: () => ({
+          helpers: {
+            status: { getCurrentPermission: () => mockStatus },
+            transaction: { getCurrentPermission: () => mockTransaction }
+          }
+        })
+      }
+      const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: true }))
+      const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+        permit: {
+          cost: 6
+        },
+        renewedEndDate: '2022-06-22T14:48:00.000Z',
+        licenceStartDate: '2022-07-22T15:48:00.000Z'
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.isContinuing).toBeFalsy()
+    })
+
+    it('should return true if a concession has been selected and only if disabled', async () => {
+      const request = {
+        cache: () => ({
+          helpers: {
+            status: { getCurrentPermission: () => mockStatus },
+            transaction: { getCurrentPermission: () => mockTransaction }
+          }
+        })
+      }
+      const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: true }))
+      const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+        permit: {
+          cost: 6
+        },
+        concessions: [
+          {
+            type: CONCESSION.DISABLED
+          }
+        ]
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.disabled).toBeTruthy()
+    })
+
+    it('should return RENEWAL_START_DATE uri if renewal is true', async () => {
+      const request = {
+        cache: () => ({
+          helpers: {
+            status: { getCurrentPermission: () => mockStatus },
+            transaction: { getCurrentPermission: () => mockTransaction }
+          }
+        })
+      }
+      const mockStatus = mockStatusCacheGet.mockImplementationOnce(() => ({ renewal: true }))
+      const mockTransaction = mockTransactionCacheGet.mockImplementationOnce(() => ({
+        licenceStartDate: mockStatus.renewal ? RENEWAL_START_DATE.uri : LICENCE_TO_START.uri,
+        licensee: { firstName: 'John', lastName: 'Smith', birthDate: '1996-01-01' },
+        numberOfRods: '3',
+        licenceLength: '12M',
+        permit: {
+          cost: 6
+        },
+        licenceType: 'Salmon and sea trout'
+      }))
+      const result = await getData(mockRequest(request))
+      expect(result.uri.licenceStartDate).toBe(RENEWAL_START_DATE.uri)
+    })
+
+    it('should return LICENCE_TO_START uri if renewal is false', async () => {
```

Comment:
'should return LICENCE_TO_START uri as uri.licenceStartDate if renewal flag isn't set